.github/workflows/ci.yml

```
@@ -331,7 +331,7 @@ jobs:
     name: 'Report results on Slack'
     runs-on: ubuntu-latest
     needs: ['project-jobs', 'project-lint-jobs', 'project-test-jobs', 'evaluate-project-jobs']
-    if: ${{ !cancelled() && github.event_name != 'pull_request' && github.repository == 'woocommerce/woocommerce'  }}
+    if: ${{ !cancelled() && github.repository == 'woocommerce/woocommerce'  }}
```

Only for testing changes. To be reverted before merging the PR

```suggestion
    if: ${{ !cancelled() && github.event_name != 'pull_request' && github.repository == 'woocommerce/woocommerce'
```